sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -55,21 +55,24 @@ def _convert_to_device(examples: torch.Tensor, device) -> torch.Tensor:
 class PytorchModelHandlerTensor(ModelHandler[torch.Tensor,
                                              PredictionResult,
                                              torch.nn.Module]):
-  """ Implementation of the ModelHandler interface for PyTorch."""
   def __init__(
       self,
       state_dict_path: str,
       model_class: Callable[..., torch.nn.Module],
       model_params: Dict[str, Any],
       device: str = 'CPU'):
-    """
-    Initializes a PytorchModelHandlerTensor
-    :param state_dict_path: path to the saved dictionary of the model state.
-    :param model_class: class of the Pytorch model that defines the model
-    structure.
-    :param device: the device on which you wish to run the model. If
-    ``device = GPU`` then a GPU device will be used if it is available.
-    Otherwise, it will be CPU.
+    """Implementation of the ModelHandler interface for PyTorch.
+
+    Example Usage:
+      pcoll | RunInference(PytorchModelHandlerTensor(state_dict_path="my_uri"))
```

Comment:
nit: I think this needs a newline and an extra colon to render as a code block, see for example https://github.com/apache/beam/blob/master/sdks/python/apache_beam/dataframe/convert.py#L50-L55

```suggestion
    Example Usage::
    
      pcoll | RunInference(PytorchModelHandlerTensor(state_dict_path="my_uri"))
```

Reply:
resolved in https://github.com/apache/beam/pull/21921